src/FlowScene.hpp

```diff
@@ -112,6 +112,8 @@ class NODE_EDITOR_PUBLIC FlowScene
 
   void nodeHoverLeft(Node& n);
 
+  void nodeContextMenu(Node& n, const QPointF& pos);
```

Comment:
Sorry, where is the implementation of this function?

Reply:
It's in the signal: section, so the implementation is moc-generated

Reply:
Oh, indeed, I overlooked it.